packages/grid/_modules_/grid/components/base/GridErrorHandler.tsx

```diff
@@ -20,10 +29,18 @@ export function GridErrorHandler(props) {
       logger={logger}
       render={(errorProps) => (
         <GridMainContainer>
-          <rootProps.components.ErrorOverlay
-            {...errorProps}
-            {...rootProps.componentsProps?.errorOverlay}
-          />
+          <GridAutoSizer
```

Comment:
Fixes https://codesandbox.io/s/datagriddemo-material-demo-forked-49plk?file=/demo.tsx

If the error is set by the `error` prop, `GridAutoSizer` in `GridBody` won't render and we won't have any dimension.